src/Storages/ColumnsDescription.h

```diff
@@ -39,7 +39,7 @@ struct ColumnDescription
     ColumnDescription() = default;
     ColumnDescription(ColumnDescription &&) = default;
     ColumnDescription(const ColumnDescription &) = default;
-    ColumnDescription(String name_, DataTypePtr type_);
+    ColumnDescription(String name_, DataTypePtr type_, String comment_ = "");
```

Comment:
I think it is better to add `comment` field to `ColumnDescription` separately instead of this change.